### ISSUE TITLE
Exporting additional modules pollutes localStorage

### ISSUE THREAD

**Issue by thisispvb** [ripeworks/local-storage-fallback#9]
You can write to `localStorage`/`sessionStorage` via the dot-notation (`localStorage.foo = 'bar';`)

When exporting the storage, the additional modules are also passed onto that storage in the same way.
```js
// src/index.js
// [...]

module.exports = storage
module.exports.isSupported = isSupported
module.exports.CookieStorage = CookieStorage
module.exports.MemoryStorage = MemoryStorage
```

Because of that the storage then ends up having the additional exports written to it as strings:

<img width="394" alt="screen shot 2017-07-25 at 16 40 12" src="https://user-images.githubusercontent.com/594622/28593105-406cd6dc-7159-11e7-9bcc-7c6f11642d76.png">

**Comment by tamagokun** [comment]
ah, of course. It seems like the only way around this would be to export a function:

```js
import createStorage from 'local-storage-fallback'

const storage = createStorage()
```

or written another way

```js
const storage = require('local-storage-fallback')()
```

OR

wrapping in an object and not exporting a default

```js
const {storage} = require('local-storage-fallback')
```

**Comment by thisispvb** [comment]
The last option sounds the best, having to call a function everywhere it is imported is tedious and not that intuitive.

If exporting via `storage` (or similar) you can also use default exports at the same time when using the `import`/`export` syntax

```js
export default storage
export { storage, isSupported, CookieStorage, MemoryStorage }
```

**Comment by tamagokun** [comment]
Agreed. I had issues before transpiling for web and ended up switching to `module.exports` but i'll give it a go.

**Comment by tamagokun** [comment]
@philipvonbargen fixed in `4.0.0` Thanks for the PR!
